update docs (#13711)
hashicorp/consul: website/content/docs/nia/enterprise/index.mdx

| Governance | | [Sentinel](https://www.terraform.io/docs/cloud/sentinel/index.html) to enforce governance policies as code |

The [Terraform Cloud driver](/docs/nia/configuration#terraform-cloud-driver) enables CTS Enterprise to integrate with Terraform Cloud or Terraform Enterprise. The [Terraform Cloud driver](/docs/nia/network-drivers/terraform-cloud) page provides an overview of how the integration works within CTS.

## Consul Admin Partition Support

> CTS subscribes to a Consul agent. Depending on the admin partition the Consul agent is a part of and the services within the admin partition, CTS will be able to subscribe to those services and support the automation workflow. As such, **admin partitions are not relevant to the CTS workflow**. The recommendation is to have a single CTS instance subscribe to services/KV within a single partition and use a different CTS instance (or instances) to subscribe to services/KV in another partition.

Reviewer: ```suggestion
CTS subscribes to a Consul agent. Depending on the admin partition the Consul agent is a part of and the services within the admin partition, CTS will be able to subscribe to those services and support the automation workflow. As such, admin partitions are not relevant to the CTS workflow. We recommend deploying a single CTS instance that subscribes to services/KV within a single partition and using a different CTS instance (or instances) to subscribe to services/KV in another partition.
```
We should be clear that the recommendation comes from HashiCorp. 
The engineering style guide doesn't have good information about emphasis and when to use bold/italics, but the education style guide recommends using plain text.

Author: Sounds good, thanks Adam
